dotcms-postman/src/main/resources/postman/Category.postman_collection.json

```diff
@@ -5402,6 +5924,22 @@
 		{
 			"key": "child3Inode",
 			"value": ""
+		},
+		{
+			"key": "topCategoryInodeHierarchy",
+			"value": ""
+		},
+		{
+			"key": "child2CategoryInodeHierarchy",
+			"value": ""
+		},
+		{
+			"key": "child3CategoryInodeHierarchy",
+			"value": ""
+		},
+		{
+			"key": "grantChildCategoryInodeHierarchy",
+			"value": ""
```

Comment:
Also let's include some negative cases for `/api/v1/categories/hierarchy` like: 
* Sending an empty body to 
* Sending bad inodes.

Reply:
done 

https://github.com/dotCMS/core/pull/29142/commits/db80122397d2846915fc160073f4c1dc27146d1a#diff-038fb4343125192a43e50d739675b735395e512d230c3b8dbeb55b1f62bd7e33R3594

https://github.com/dotCMS/core/pull/29142/commits/db80122397d2846915fc160073f4c1dc27146d1a#diff-038fb4343125192a43e50d739675b735395e512d230c3b8dbeb55b1f62bd7e33R3529